include/envoy/tracing/http_tracer.h

```diff
@@ -67,6 +67,8 @@ class HttpTracer {
 public:
   virtual ~HttpTracer() {}
 
+  virtual void installDriver(DriverPtr&& driver) PURE;
```

Comment:
I see why you need this in the config code, but I would prefer that we not expose this function around the code base. If you need it to be part of the interface, can we make another interface the derives from this interface that allows driver installation?